README.md

```diff
@@ -261,17 +261,21 @@ Aggie uses Google Places for guessing locations in the application. To make it w
 1. Set `fetching` value to enable/disable fetching for all sources at global level.
   - This is also changed during runtime based on user choice.
 
+
 ### Logging
 
-Set various logging options in `logger` section:
+Set various logging options in `logger` section.
 
-- `console` section is for console logging. For various options, see [winston](see https://github.com/winstonjs/winston#transports)
```

Comment:
I would love this to actually work. For development it's really useful, and with PM2 in production the console logs automatically get written to file anyway.

Do you think it'd be much work to add it (or does it already log to console but it's not configurable?)

Reply:
I would too! It's really confusing for nothing to be logged while trying to work from a development instance. Happy to look into it...

Reply:
@cooperka Requested changes made!

- Disabled field is now at the top for every transport section in `config/secrets.json.example`
- Added the console transport for *all* log levels and enable by default

Fun fact: the NPM log level standard has a special level called `"silly"` for when you want to log something but feel probably embarrassed doing it :laughing:

Reply:
`logger.silly("Day 93. I started watching Love Island on Hulu. It's surprisingly interesting...");`